services/mozilla-observatory/mozilla-observatory.service.js

```diff
@@ -46,42 +46,32 @@ const documentation = `
 `
 
 module.exports = class MozillaObservatory extends BaseJsonService {
-  static get category() {
-    // TODO: Once created, change to a more appropriate category,
-    // see https://github.com/badges/shields/pull/2926#issuecomment-460777017
```

Comment:
Let's be sure that we don't drop the inline comment as part of the refactoring

Reply:
Sure will keep this in mind.Thanks

Reply:
No worries. This should be good to go once you restore this comment

Reply:
@calebcartwright changes done.